target/linux/ath79/nand/target.mk

```diff
@@ -1,9 +1,10 @@
 BOARDNAME := Generic devices with NAND flash
-FEATURES += squashfs nand rtc
+
+FEATURES += squashfs nand
 
 DEFAULT_PACKAGES += wpad-basic
 
 define Target/Description
-	Build firmware for Atheros AR71xx/AR913x based boards with
-	NAND flash, e.g. Netgear WNDR4300.
+	Firmware for boards based on MIPS 24kc Atheros/Qualcomm SoCs
```

Comment:
Not all QCA SoCs are mips24kc. ar9341 and qca9558 are actually mips 74kc chips. Drop "24kc" here and maybe consider changing "Atheros/Qualcomm" to "Qualcomm Atheros" instead.